_data/simple-icons.json

```diff
@@ -355,6 +355,11 @@
             "hex": "2A73CC",
             "source": "https://about.coursera.org/press"
         },
+        {
+            "title": "Creative Cloud",
```

Comment:
This should be "Adobe Creative Cloud", similar to how Photoshop is "Adobe Photoshop". Remember to update the SVG too :)